### DISCUSSION TITLE
Find Inactives

### DISCUSSION THREAD

**Discussion by kathrynbizzz** [AprilSylph/XKit-Rewritten#301] [Suggestion Box]
Hi, I was wondering if there were any plans in the future to add the "Find Inactives" widget. I know tumblr has a feature that will allow you to see when the person was last active, but I really miss being able to mass unfollow blogs that haven't posted in over a specific amount of time. 

Thank you in advance for taking the time to read this! 

- KD

**Comment by AprilSylph** [comment]
Yes, this is planned. https://github.com/AprilSylph/XKit-Rewritten/projects/4

**Comment by Grerutin** [comment]
glad i checked the discussions to see if anyone else was thinking the same thing. last time that particular xkit feature worked for me was a little after the nsfw purge, so its been years since I've been able to unfollow inactive users. I've just been hopping around to random numbers in the url and checking manually haha

**Comment by ethanhines** [comment]
is this an important feature to implement I am following 5,000 blogs (the max) and the only way I can find inactive is go though one by one to see when the last post was made tldr == ANNOYING

**Comment by yxngtrxll** [comment]
Hi, just wondering if there is any update on when this planned project will be looked into? It's definitely top of my list in terms of what new features I would love to see added. hopefully others are the same. PS, thank you for this incredible extension. I would be lost without it. Everyone who has helped make this program possible and continues to work on it has my utmost respect and many thanks!
